components/notices/ChainDeprecation.tsx

```diff
@@ -0,0 +1,32 @@
+import { useActiveChainId } from "@3rdweb-sdk/react";
+import { Flex } from "@chakra-ui/react";
+import { SUPPORTED_CHAIN_ID } from "@thirdweb-dev/sdk";
+import { deprecatedChains } from "constants/mappings";
+import { Text } from "tw-components";
+
+export const ChainDeprecation = () => {
+  const currentDate = new Date();
+  const deprecationDate = new Date("2021-10-05T00:00:00.000Z");
```

Comment:
This will work but I would typically set it up outside of the component so it doesn't get re-instantiated on every render